logstreams/src/main/java/io/zeebe/logstreams/impl/log/LogStreamImpl.java

```diff
@@ -144,6 +149,38 @@ private void internalSetCommitPosition(final long commitPosition) {
     onCommitPositionUpdatedConditions.signalConsumers();
   }
 
+  @Override
+  public void delete(final long position) {
+    actor.call(
```

Comment:
Out of curiosity, why `call` and not `run` if we don't return anything? Doesn't it just wrap the runnable in a `Callable<Void>`?

Reply:
I moved the method yes. In the past we had the pattern if there comes an request from outside we use call for the context switch. The run method was later improve to handle this as well.